composer.json

```diff
@@ -22,10 +22,14 @@
         "source": "https://github.com/portphp/doctrine",
         "docs": "https://portphp.readthedocs.org"
     },
+    "minimum-stability": "dev",    
```

Comment:
I think this kind of configuration should go to the bottom of the file: most people are rather interested in requirements than stability settings for development.